diff --git a/src/components/Button.svelte b/src/components/Button.svelte
new file mode 100644
--- /dev/null
+++ b/src/components/Button.svelte
@@ -0,0 +1,61 @@
+<script lang="ts">
+  /**
+   * Is this the principal call to action on the page?
+   */
+  export let primary = false;
+
+  /**
+   * What background color to use
+   */
+  export let backgroundColor: string | undefined = undefined;
+  type buttonSize = 'small' | 'medium' | 'large';
+  /**
+   * How large should the button be?
+   */
+  export let size: buttonSize = 'medium';
+  /**
+   * Button contents
+   */
+  export let label = '';
+
+  $: sizeClasses = (size: buttonSize) => {
+    let classes: string[] = [];
+
+    switch (size) {
+      case 'small':
+        classes = ['text-xs', 'py-2', 'px-4'];
+        break;
+      case 'large':
+        classes = ['text-base', 'py-3', 'px-6'];
+        break;
+      case 'medium':
+      default:
+        classes = ['text-sm', 'py-2.5', 'px-5'];
+        break;
+    }
+    return classes;
+  };
+
+  const common = [
+    'font-sans',
+    'font-bold',
+    'border-0',
+    'rounded-3xl',
+    'cursor-pointer',
+    'inline-block',
+    'line-height',
+    'leading-none',
+  ];
+
+  $: style = primary ? '' : 'box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 1px inset;';
+
+  $: bgOverride = `bg-${backgroundColor}`;
+  $: mode = primary ? ['text-white', 'bg-[#1ea7fd]'] : ['text-[#333]', 'bg-transparent'];
+  $: classAttr = [...common, ...mode, bgOverride, ...sizeClasses(size)].join(
+    ' '
+  );
+</script>
+
+<button type="button" class={classAttr} {style} on:click>
+  {label}
+</button>
diff --git a/src/stories/Button.stories.ts b/src/stories/Button.stories.ts
--- a/src/stories/Button.stories.ts
+++ b/src/stories/Button.stories.ts
@@ -1,6 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/svelte';

-import Button from './Button.svelte';
+import Button from './components/Button.svelte';

 // More on how to set up stories at: https://storybook.js.org/docs/7.0/svelte/writing-stories/introduction
 const meta = {
diff --git a/src/stories/Button.svelte b/src/stories/Button.svelte
deleted file mode 100644
--- a/src/stories/Button.svelte
+++ /dev/null
@@ -1,34 +0,0 @@
-<script lang="ts">
-  import './button.css';
-
-  /**
-   * Is this the principal call to action on the page?
-   */
-  export let primary = false;
-
-  /**
-   * What background color to use
-   */
-  export let backgroundColor: string | undefined = undefined;
-  /**
-   * How large should the button be?
-   */
-  export let size: 'small' | 'medium' | 'large' = 'medium';
-  /**
-   * Button contents
-   */
-  export let label = '';
-
-  $: mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
-
-  $: style = backgroundColor ? `background-color: ${backgroundColor}` : '';
-</script>
-
-<button
-  type="button"
-  class={['storybook-button', `storybook-button--${size}`, mode].join(' ')}
-  {style}
-  on:click
->
-  {label}
-</button>
diff --git a/src/stories/Header.svelte b/src/stories/Header.svelte
--- a/src/stories/Header.svelte
+++ b/src/stories/Header.svelte
@@ -1,6 +1,6 @@
 <script lang="ts">
   import './header.css';
-  import Button from './Button.svelte';
+  import Button from './components/Button.svelte';

   import { createEventDispatcher } from 'svelte';

diff --git a/src/stories/button.css b/src/stories/button.css
deleted file mode 100644
--- a/src/stories/button.css
+++ /dev/null
@@ -1,30 +0,0 @@
-.storybook-button {
-  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-  font-weight: 700;
-  border: 0;
-  border-radius: 3em;
-  cursor: pointer;
-  display: inline-block;
-  line-height: 1;
-}
-.storybook-button--primary {
-  color: white;
-  background-color: #1ea7fd;
-}
-.storybook-button--secondary {
-  color: #333;
-  background-color: transparent;
-  box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 1px inset;
-}
-.storybook-button--small {
-  font-size: 12px;
-  padding: 10px 16px;
-}
-.storybook-button--medium {
-  font-size: 14px;
-  padding: 11px 20px;
-}
-.storybook-button--large {
-  font-size: 16px;
-  padding: 12px 24px;
-}
diff --git a/src/stories/components b/src/stories/components
new file mode 120000
--- /dev/null
+++ b/src/stories/components
@@ -0,0 +1 @@
+../components/
\ No newline at end of file
